application1.py

```diff
@@ -0,0 +1,73 @@
+import csv
+
+book_file = open("books.csv",'rt')
+book_names = csv.reader(book_file)
+
+author_file = open("authors.csv",'rt')
+author_names = csv.reader(author_file)
+
+
+def main():
+    book_list = []
+    count = 0
+    another_entry = 1
+
+    print("----------Welcome to the book store!----------")
+    print("________________________________________________\n")
+    
+    print("Here's our catalogue to explore around:")
+    print("""
+    1. Books
+    2.Authors
+    3.Book Reviews
+    """)
+
+    choice = int(input("Enter your choice: "))
+
+    if choice ==1:
+        print("""
+1. Browse through the book catalogue.
+2. Add books to the catalogue.
+		""")
+		
+        user_input = int(input("Enter your choice as '1' or '2': "))
+        
+        if user_input == 1:
+            print("Here's our Book catalogue:")
+            
+            for row in book_names:
+                print("|\t",row[0],"|\t", row[1],"|\t", row[2],"|\t", row[3])
```

Comment:
convert this into a Book object. Then add a method inside the Book object to print it.

Reply:
```book = Book(id=row[0]. ...)
book.print()```